samples/highcharts/accessibility/accessible-table/demo.js

```diff
@@ -26,12 +26,18 @@ Highcharts.chart('container', {
     },
     series: [{
         name: 'John',
-        data: [5, 3, 4, 7, 2]
+        data: [5, 3, 4, 7, 2],
+        borderWidth: 2,
+        borderColor: '#000000'
     }, {
         name: 'Jane',
-        data: [2, 2, 3, 2, 1]
+        data: [2, 2, 3, 2, 1],
+        borderWidth: 2,
+        borderColor: '#000000'
     }, {
         name: 'Joe',
-        data: [3, 4, 4, 2, 5]
+        data: [3, 4, 4, 2, 5],
+        borderWidth: 2,
+        borderColor: '#000000'
     }]
 });
```

Comment:
It can, but doesn't have to be. I think the initial thought of this was that when the demo is bigger then usual(since the table is open), it is nice to place the description at the top so it becomes more visible.